r_markdown/carbonyls.Rmd

```diff
@@ -123,7 +123,8 @@ Load notes
                                     -id, -id_ions, -flow,
                                     -start, -end, -type, -qc) %>%
                       dplyr::filter(!is.na(pol)) %>%
-                      dplyr::mutate(pol = as.factor(pol))
+                      dplyr::mutate(pol = as.factor(pol)) %>%
+                      dplyr::select(-id_ions)
```

Comment:
- I just don't like that the .Rda file is called "ions" and "id_ions" in the carbonyl file...wasn't there something in the style guide about using appropriate variable names?

- okay, okay I guess ions are really ions + anhydrosugars and other carbohydrates (all analyzed on the same filter)